packages/babel-core/src/config/index.js

```diff
@@ -3,7 +3,6 @@
 import path from "path";
 import * as context from "../index";
 import Plugin from "./plugin";
-import merge from "lodash/merge";
```

Comment:
Once we make dependencies lazy-load, not loading lodash during anywhere in `src/config` makes a big difference.